app/views/users/_users_table.html.slim

```diff
@@ -3,30 +3,30 @@
     .col-md-12
       h4 = "#{user_type.upcase} TABLE"
   .row
-    .col-md-3
-      h5 Firstname
-    .col-md-3
-      h5 Lastname
-    .col-md-3
-      h5 Email
-    .col-md-3
+    .col-md-3.user-info
```

Comment:
@RedzepovicSanin When `I18n` does not have translation it will translate symbol used to String. These translations work for that reason but they are not used from `locales/en.yml` file. Make sure you use full translation path `users.firstname` for them to work.

Also, `PRO TIP` as project gets bigger `en.yml` file we grow and grow at point where it is couple of thousand lines translation file. To avoid that create separate translation files in `locales` folder. For users it would be `locales/users.yml` with same content `en.yml` has now.